plugins/oomlie.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: oomlie
+spec:
+  version: 'v0.0.1'
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: "https://github.com/jdockerty/kubectl-oomlie/releases/download/v0.0.1/oomlie_linux_amd64.tar.gz"
+    sha256: "3aa2cc916a2a13a54556752632b7f03df0869dd9d0f4e7007502ebcb1925de5d"
+    files:
+    - from: "./oomlie"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "oomlie"
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: "https://github.com/jdockerty/kubectl-oomlie/releases/download/v0.0.1/oomlie_darwin_amd64.tar.gz"
+    sha256: "a67f3bbd4de13e90e208cc7039557dafe42e2d84fc3b4ddd09a9adf5dc180238"
+    files:
+    - from: "./oomlie"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "oomlie"
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: "https://github.com/jdockerty/kubectl-oomlie/releases/download/v0.0.1/oomlie_windows_amd64.zip"
+    sha256: "32c3a1bc1091385b7bab0c85888e93805a3352d9fc548f9542fb59fb87368ecf"
+    files:
+    - from: "/oomlie.exe"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "oomlie.exe"
+  shortDescription: Show pods which have recently been OOMKilled
```

Comment:
> Show recently OOMKilled Pods

Reply:
Much more concise, I like it :rocket: 

Changing this now :blush: